Commands/Search/GetSiteHistoricalVersionsStatus.cs

```diff
@@ -0,0 +1,56 @@
+﻿#if !ONPREMISES
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.SharePoint.Client;
+using Microsoft.SharePoint.Client.Search.Administration;
+using PnP.PowerShell.CmdletHelpAttributes;
+
+namespace PnP.PowerShell.Commands.Search
+{
+    [Cmdlet(VerbsCommon.Get, "PnPSiteHistoricalVersionsStatus", DefaultParameterSetName = "Xml")]
+    [CmdletHelp("Returns information about the Historical Versions feature for the current site collection from the context. " +
+       "The document statistics are only updated periodically, check the next update time property (in UTC) to see when new data will be available.",
+        SupportedPlatform = CmdletSupportedPlatform.Online,
+        Category = CmdletHelpCategory.Search)]
+    [CmdletExample(
+        Code = @"PS:> Get-PnPSiteHistoricalVersionsStatus",
+        Remarks = "Returns the status of the feature as well as the number of documents processed for the site if the feature is enabled.",
+        SortOrder = 1)]
+    public class GetSiteHistoricalVersionsStatus : PnPWebCmdlet
+    {
```

Comment:
@erwinvanhunen would it make sense to inherit from `PnPAdminCmdlet` instead, and maybe move the cmdlet to the Admin folder instead of search?

Reply:
It probably would make more sense indeed. What's the current status of the availability of this API? Does anyone know? Still doesn't seem to compile here against the latest CSOM version.

Reply:
We have no wait until the CSOM change is pushed. Don't have an ETA, but maybe @VesaJuvonen has track of this?